packages/database/zod/insights.ts

```diff
@@ -1,30 +1,33 @@
+import { type Insight, InsightCategory } from "@prisma/client";
 import { z } from "zod";
-import { ZId } from "./common";
+import { ZId } from "../../types/common";
 
-export const ZInsightCategory = z.enum(["featureRequest", "complaint", "praise", "other"]);
+export const ZInsightCategory = z.nativeEnum(InsightCategory);
 
-export type TInsightCategory = z.infer<typeof ZInsightCategory>;
-
-export const ZInsight = z.object({
+const InsightModel = z.object({
   id: ZId,
   createdAt: z.date(),
   updatedAt: z.date(),
   environmentId: ZId,
   title: z.string(),
   description: z.string(),
   category: ZInsightCategory,
+}) satisfies z.ZodType<Insight>;
+
+export const ZInsight = InsightModel.extend({
```

Comment:
when is `ZInsight` used? It includes another table and thus shouldn't be part of this type. We only want to include the fields that are actually present in the insight model.